ChangeLog.md

```diff
@@ -23,10 +23,17 @@ Major changes:
 
 * Fully remove GHCJS support.
 
+* Remove the `freeze` command. It has been replaced by lock files.
```

Comment:
Is this somehow related to generated cabal files?

Reply:
Yes, I explained it in a commit message, but didn't want to waste space in the changelog. Since there's a change in the desired semantics of freeze, we could either introduce that change, _or_ just get rid of the command. Since we want to get rid of the command anyway, may as well just do it now.

Reply:
Thanks